content/300-guides/500-other/900-advanced-database-tasks/06-typesafe-raw-sql/01-safeql-gis.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Using SafeQL with unsupported GIS features'
+metaTitle: 'Using SafeQL with unsupported GIS features (Guide)'
+metaDescription: 'Learn how to use SafeQL and Prisma Client extensions to work around features not natively supported by Prisma.'
+---
+
+## Overview
+
+This page explains how to use Prisma Client extensions and [SafeQL](https://safeql.dev) in order to make the experience of working with geographic data better.
+
+Our example will be using [PostGIS](https://postgis.net/) and PostgreSQL, but is applicable to other features and database engines.
+
+## Prerequisites
+
+To follow this guide, we recommend:
+
+- an existing [PostgreSQL](https://www.postgresql.org/) database
+- an existing Prisma project
+
+## What is SafeQL?
+
+[SafeQL](https://safeql.dev/) is an ESLint plugin that allows for advanced linting and type safety within raw SQL queries. After setup, SafeQL works with Prisma `$queryRaw` and `$executeRaw` to provide type safety when raw queries are required.
+
+## Geographic data support in Prisma
+
+At this time geographic data, and specifically PostGIS, are not supported by Prisma. Models that have geographic data columns will not have the benefit of type safety, nor will be able to use mutating operations like `create` and `update`.
```

Comment:
```suggestion
At the time of writing, Prisma does not support working with geographic data, specifically using PostGIS. 

A model that has geographic data columns will be stored using the [`Unsupported`](/reference/api-reference/prisma-schema-reference#unsupported) data type. Fields with `Unsupported` types are present in the generated Prisma Client. A model with a required `Unsupported` type does not expose write operations such as `create`, and `update`.
```

Reply:
https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#unsupported (copy-pasta-d from this page)